src/hooks/useGetConversations.js

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useCallback } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useStatus } from '@rootstrap/redux-tools';
+import { getConversations } from 'actions/chatActions';
+
+const useGetConversations = navigation => {
+  const dispatch = useDispatch();
+  const requestConversations = useCallback(() => {
+    dispatch(getConversations());
+  }, [dispatch]);
```

Comment:
do you need the brackets here? or does this work? 
`const requestConversations = useCallback(() => dispatch(getConversations()), [dispatch]);`